docs/guides/connecting_to_your_data/overview.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Connect to data: Overview"
+---
+# [![Connect to data icon](../../images/universal_map/Outlet-active.png)](overview.md) Connect to data: Overview 
+
+import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<!--Use 'inactive' or 'active' to indicate which Universal Map steps this term has a use case within.-->
+
+<UniversalMap setup='inactive' connect='active' create='inactive' validate='inactive'/>
+
+<!-- Only keep one of the 'To best understand this document' lines.  For processes like the Universal Map steps, use the first one.  For processes like the Architecture Reviews, use the second one. -->
+
+:::note Prerequisites
+- Completing [Step 2: Connect to data](/docs/tutorials/getting_started/connect_to_data) of the Getting Started tutorial is recommended.
+:::
+	
+Connecting to your data in Great Expectations is designed to be a painless process.  Once you have performed this step, you will have a consistent API for accessing and validating data on all kinds of source data systems: SQL-type data sources, local and remote file stores, in-memory data frames, and more.
+
+## The connect to data process
+
+<!-- Brief outline of what the process entails.  -->
+
+Connecting to your data is built around the Datasource object.  A Datasource provides a standard API for accessing and interacting with data from a wide variety of source systems.  It does this by providing an interface for a Data Connector and an Execution Engine to work together.
```

Comment:
Should we throw in a quick description about what Data Connectors and Execution Engines do here? Otherwise the reference might be confusing.

Reply:
I agree with Don.  And those tags should be included very soon -- I may even add them to this PR just to address this question, and so they are available to use in future documentation as it is composed.